### PR TITLE
MM-10652 - implemented reboot command

### PR DESCRIPTION
**Pull request by reflog**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -62,3 +62,5 @@ sessions.db
 loadtestconfig.json
 
 /terraformenv
+.idea
+mattermost-load-test.iml
```
Comment (lieut-data) [inline .gitignore]: nit: missing trailing newline? .editorconfig not yet setup?

**Review by lieut-data — APPROVED** [review]
This seems fine, but I wonder if instead of a dedicated `reboot` command we just added a dedicated `ssh --instance=all` or some such? I often find myself wrapping an arbitrary ssh command in a bash for loop.

**Comment by reflog** [comment]
> This seems fine, but I wonder if instead of a dedicated `reboot` command we just added a dedicated `ssh --instance=all` or some such? I often find myself wrapping an arbitrary ssh command in a bash for loop.

@lieut-data  I think  `ssh --instance=all` is a great idea, but let's open a separate ticket for it?

File: .gitignore (hunk 1 above)
Comment (reflog) [reply .gitignore]: I'll add .editorconfig and fix

**Review by crspeller — APPROVED** [review]

**Pull request merged**